gulpfile.js

```diff
@@ -63,11 +63,6 @@ var buildDist = function(opts) {
       'react': 'React',
       'react-dom': 'ReactDOM'
     },
-    module: {
-      loaders: [
-        {test: /\.js$/, loader: 'babel'}
-      ],
-    },
```

Comment:
Ok, I'll come clean…

The real reason I found this was because I couldn't build relay after making changes to fbjs because babel didn't like transforming the code it generated and I got this:

```
ERROR in Loader /Users/poshannessy/FB/code/fbjs/node_modules/babel/index.js didn't return a function
 @ ../fbjs/lib/fetchWithRetries.js 26:27-60
```